### PR TITLE
Minilm

### PR DESCRIPTION
**Pull request by thiagolaitz**
Adds miniLM support in evaluate_kaggle_highlighter.py and updates s.string to s.text in tokenize.py

### CODE REVIEW

**Review by rodrigonogueira4 — APPROVED** [review]
LGTM! @ronakice , could you please take a look and merge it once you are happy?

File: pygaggle/run/evaluate_kaggle_highlighter.py [hunk 1]
```diff
@@ -6,7 +6,8 @@
 from transformers import (AutoModel,
```
Comment (ronakice) [inline pygaggle/run/evaluate_kaggle_highlighter.py:6]: Can you move AutoModel to the next line?

File: pygaggle/run/evaluate_kaggle_highlighter.py [hunk 2]
```diff
@@ -131,6 +133,13 @@ def construct_qa_transformer(options: KaggleEvaluationOptions) -> Reranker:
                     use_fast=False)
     return QuestionAnsweringTransformerReranker(model, tokenizer)
 
+  
+def construct_minilm(options: KaggleEvaluationOptions) -> Reranker:
+    if options.model != None:
```
Comment (ronakice) [inline pygaggle/run/evaluate_kaggle_highlighter.py:138]:
```suggestion
    if options.model:
```

**Review by ronakice — CHANGES_REQUESTED** [review]
Two small comments, thanks for adding this! Once these have been addressed we can merge!

File: pygaggle/run/evaluate_kaggle_highlighter.py (hunk 1 above)
Comment (ronakice) [inline pygaggle/run/evaluate_kaggle_highlighter.py:6]:
```suggestion
from transformers import (
AutoModel,
```

File: pygaggle/model/tokenize.py [hunk 3]
```diff
@@ -176,8 +176,8 @@ class SpacySenticizer:
 
     def __init__(self, max_paragraph_length: int = None):
         self.max_paragraph_length = max_paragraph_length
-
+           
     @lru_cache(maxsize=1024)
     def __call__(self, document: str) -> List[str]:
-        return [s.string for s in self.nlp(
+        return [s.text for s in self.nlp(
             document[:self.max_paragraph_length]).sents]
```
Comment (ronakice) [inline pygaggle/model/tokenize.py:183]:
```suggestion
            document[:self.max_paragraph_length]).sents]
            
```

**Review by ronakice — APPROVED** [review]
Fixed some style stuff, merging!

**Pull request merged**